SPECS/wget/wget.spec

```
@@ -61,6 +66,9 @@ make  %{?_smp_mflags} check
 %{_datadir}/locale/*/LC_MESSAGES/*.mo
 
 %changelog
+* Thu Sep 28 2023 Osama Esmail <osamaesmail@microsoft.com> - 
```

We might want to bump the `release` tag as well:
```suggestion
* Thu Sep 28 2023 Osama Esmail <osamaesmail@microsoft.com> - 1.21.2-2
```